distribution/docker/src/docker/Dockerfile

```diff
@@ -364,6 +369,9 @@ LABEL name="Elasticsearch" \\
 RUN mkdir /licenses && \\
     cp LICENSE.txt /licenses/LICENSE
 <% } %>
+<% if (docker_base == 'iron_bank') { %>
+COPY LICENSE /licenses/LICENSE.addendum
```

Comment:
https://github.com/elastic/ironbank/blob/master/Elastic_License_w_gov_addendum_2020-10-08-1.txt contains standard license + addendum so it should completely replace the other license file.

Can you also add this license file to the generated build context tarball?

Reply:
The simplest thing to do would be to copy that file into the ES repo and add it into the generated build context. What do you think? Obviously having it in two places isn't ideal, but fetching the license file when the task run also feels fragile.

Reply:
I totally agree 👍

Reply:
The license is now included in the repo and in the generated build context.